### PR TITLE
Mark all failed ONNX layer tests as skip

### PR DESCRIPTION
**Pull request by dkrupnov**
### Details:
Mark all failed ONNX layer tests as xfail

Idea is to have a current snapshot as accepted and just monitor for regressions.

### CODE REVIEW

**Comment by vurusovs** [comment]
Hello!

Is there any explanation why tests start failing? And why it is expected? 
It will be required when tests need to be corrected

**Comment by dkrupnov** [comment]
Hello @vurusovs !

We decided with @andrei-kochin, @mlukasze and @mbencer to mark all failed ONNX layer tests as xfail and watch only new appeared issues.

**Comment by andrei-kochin** [comment]
@vurusovs I've added to PR description the main idea. Also discussed with @dkrupnov to have tests skipped with comment GREEN_SUITE

File: tests/layer_tests/onnx_tests/test_add_sub_mul_div.py [hunk 1]
```diff
@@ -298,55 +298,63 @@ def test_div_const_precommit(self, params, ie_device, precision, ir_version, tem
 
     @pytest.mark.parametrize("params", test_data)
     @pytest.mark.nightly
+    @pytest.mark.xfail(reason='Mark as xfailed all failed onnx layer tests')
```
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_add_sub_mul_div.py:301]: Can you please create a ticket and assign it to the ONNX team?

**Review by rkazants — APPROVED** [review]

**Pull request merged**